src/sync/pull-requests-new.test.ts

```diff
@@ -0,0 +1,402 @@
+				starred_url: "https://api.github.com/users/reviewer/starred{/owner}{/repo}",
+				subscriptions_url: "https://api.github.com/users/reviewer/subscriptions",
+				organizations_url: "https://api.github.com/users/reviewer/orgs",
+				repos_url: "https://api.github.com/users/reviewer/repos",
+				events_url: "https://api.github.com/users/reviewer/events{/privacy}",
+				received_events_url: "https://api.github.com/users/reviewer/received_events",
+				type: "User",
+				site_admin: false
+			},
+			body: "Here is the body for the review.",
+			state: "APPROVED",
+			html_url: "https://github.com/integrations/test-repo-name/pull/1#pullrequestreview-80",
+			pull_request_url: "https://api.github.com/repos/integrations/test-repo-name/pulls/1",
+			_links: {
+				html: {
+					href: "https://github.com/integrations/test-repo-name/pull/1#pullrequestreview-80"
+				},
+				pull_request: {
+					href: "https://api.github.com/repos/integrations/test-repo-name/pulls/1"
+				}
+			},
+			submitted_at: "2019-11-17T17:43:43Z",
+			commit_id: "ecdd80bb57125d7ba9641ffaa4d7d2c19d3f3091",
+			author_association: "COLLABORATOR"
+		}
+	];
+
+	const buildJiraPayload = (repoId: string, times = 1) => {
+		const pr = {
+			"author": {
+				"avatar": "test-pull-request-author-avatar",
+				"name": "test-pull-request-author-login",
+				"email": "test-pull-request-author-login@noreply.user.github.com",
+				"url": "test-pull-request-author-url"
+			},
+			"commentCount": 10,
+			"destinationBranch": "devel",
+			"destinationBranchUrl": "https://github.com/integrations/test/tree/devel",
+			"displayId": "#51",
+			"id": 51,
+			"issueKeys": [
+				"TES-15" // "KEY-15"
+			],
+			"lastUpdate": "2018-05-04T14:06:56Z",
+			"reviewers": [
+				{
+					"avatar": "test-pull-request-reviewer-avatar",
+					"name": "test-pull-request-reviewer-login",
+					"email": "test-pull-request-reviewer-login@email.test",
+					"url": "https://github.com/reviewer",
+					"approvalStatus": "APPROVED"
+				}
+			],
+			"sourceBranch": "use-the-force",
+			"sourceBranchUrl": "https://github.com/integrations/test/tree/use-the-force",
+			"status": "DECLINED",
+			"timestamp": "2018-05-04T14:06:56Z",
+			"title": "[TES-15] Evernote Test",//"[KEY-15] Testing force pushes"
+			"url": "https://github.com/integrations/test/pull/51",
+			"updateSequenceId": 12345678
+		};
+
+		return {
+			"preventTransitions": true,
+			operationType: "BACKFILL",
+			"repositories":
+				[
+					{
+						"id": repoId,
+						"name": "test-repo-name",
+						"pullRequests": Array(times).fill(pr),
+						"url": "test-repo-url",
+						"updateSequenceId": 12345678
+					}
+				],
+			"properties":
+				{
+					"installationId": DatabaseStateCreator.GITHUB_INSTALLATION_ID
+				}
+		};
+	};
+
+	describe("cloud", () => {
+
+		// let repoSyncState: RepoSyncState;
+
+		const PRS_INITIAL_CURSOR = 21;
+
+		beforeEach(async () => {
+			// repoSyncState = (await new DatabaseStateCreator()
```

Comment:
Lots of commented out code around these parts 🙃

Reply:
ah eww, gone!